spec/factories/submission_factories.rb

```diff
@@ -67,7 +71,16 @@
     factory :library_order do
       assets { create_list :untagged_well, 1 }
       request_types { [create(:library_request_type).id] }
-      request_options { { fragment_size_required_from: 100, fragment_size_required_to: 200, library_type: 'Standard' } }
+      request_options do
+        {
+          fragment_size_required_from: 100,
+          fragment_size_required_to: 200,
+          library_type: 'Standard',
+          bait_library_name: 'Bait'
```

Comment:
Why is bait_library_name needed now?

Reply:
Because this is where it gets the data from for `it 'has a bait_library'` test in library_complete_spec and pool_released_spec